contacts.js

```diff
@@ -3,25 +3,28 @@ const isFeed = require('ssb-ref').isFeed
 // track contact messages, follow, unfollow, block
 
 module.exports = function (sbot, createLayer) {
-  const layer = createLayer('contacts')
+  const updateLayer = createLayer('contacts')
```

Comment:
It was a bit confusing that `layer` was a function used to update the layer, so I just called it `updateLayer`